server/src/main/scala/services/exports/summaries/flights/TerminalFlightsWithActualApiSummary.scala

```diff
@@ -36,8 +36,18 @@ case class TerminalFlightsWithActualApiSummary(flights: Seq[ApiFlightWithSplits]
 }
 
 object TerminalFlightsWithActualApiSummary {
-  def actualApiHeadingsForFlights(flights: Seq[ApiFlightWithSplits]): Seq[String] =
-    flights.flatMap(f => Exports.actualAPISplitsAndHeadingsFromFlight(f).map(_._1)).distinct.sorted
+  def actualApiHeadingsForFlights(flights: Seq[ApiFlightWithSplits]): Seq[String] = Seq(
```

Comment:
Let's put these titles in the `PaxTypeAndQueue` class like we've done with the keys for the form. We can then just list the `PaxTypeAndQueue`s, mapping over them to add the 'API Actual' prefix

We can then also use them to produce the export headings and avoid any mismatch issues if they are renamed in the future